attributes/default.rb

```diff
@@ -325,7 +325,7 @@
                                              libical-devel postgresql-devel postgresql-server sendmail libxml2-devel libglvnd-devel
                                              mdadm python python-pip libssh2-devel libgcrypt-devel libevent-devel glibc-static bind-utils
                                              iproute NetworkManager-config-routing-rules python3 python3-pip iptables libcurl-devel yum-plugin-versionlock
-                                             coreutils moreutils)
+                                             coreutils moreutils sssd sssd-tools sssd-ldap curl)
```

Comment:
curl wasn't installed before?

Reply:
Probably curl is in the vanilla base AMI?